lerna.json

```diff
@@ -1,15 +1,12 @@
 {
-  "packages": [
-    "packages/*"
-  ],
+  "packages": ["packages/*"],
   "npmClient": "yarn",
-  "useWorkspaces": true,
-  "useNx": true,
   "version": "1.11.1",
-  "stream": "true",
+  "stream": true,
   "command": {
     "version": {
       "message": "chore(release): %s"
     }
-  }
+  },
+  "$schema": "node_modules/lerna/schemas/lerna-schema.json"
```

Comment:
why remove the schema?

Reply:
It's added, not removed. There is a command `lerna repair` which formats and fix the lerna config file automatically.